org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/SourceAssistProcessor.java

```diff
@@ -134,6 +140,19 @@ private TextEdit getOrganizeImportsProposal(IInvocationContext context) {
 		return null;
 	}
 
+	private Optional<Either<Command, CodeAction>> getOrganizeImportsAction(CodeActionParams params) {
+		Command command = new Command(CorrectionMessages.ReorgCorrectionsSubProcessor_organizeimports_description, COMMAND_ID_ACTION_ORGANIZEIMPORTS, Collections.singletonList(params));
+		if (preferenceManager.getClientPreferences().isSupportedCodeActionKind(CodeActionKind.SourceOrganizeImports)) {

```

Comment:
at this point, since this is a private method that's not used elsewhere, you know isSupportedCodeActionKind==true so no need to retest it.